### DISCUSSION TITLE
Add a shortcut to rotate the brush angle

### DISCUSSION THREAD

**Discussion by GFTKgithub** [photopea/photopea#6168] [Ideas]
It's a really small feature, but it would make a tremendous positive impact on the fluidity of the work.

In photoshop the shortcut is shift + arrows I believe

**Comment by photopea** [comment]
wow, I did not know about this feature! It is implemented now :)

**Reply by GFTKgithub** [reply]
Sorry for the late response.... Thank you! I'm amazed by the engagement with your users. 
I would also love if the shortcut could be Shift + ScrollWheel rather than arrows, or both.
